horovod/keras/__init__.py

```diff
@@ -108,6 +110,79 @@ def DistributedOptimizer(optimizer, name=None,
         groups=groups,
     )
 
+def PartialDistributedOptimizer(optimizer, name=None,
+                         device_dense='', device_sparse='',
```

Comment:
Nit: indentation is a bit inconsistent here and in the function body. (One pass of "Reformat Code" in PyCharm over the function should clean it up, for instance)